esrally/track/params.py

```diff
@@ -802,9 +898,12 @@ def __init__(self, index_name, type_name, conflicting_ids=None, conflict_probabi
             self.meta_data_index_with_id = '{"index": {"_index": "%s", "_id": "%s"}}\n' % (index_name, "%s")
             self.meta_data_update_with_id = '{"update": {"_index": "%s", "_id": "%s"}}\n' % (index_name, "%s")
             self.meta_data_index_no_id = '{"index": {"_index": "%s"}}\n' % index_name
-
+            self.meta_data_create_no_id = '{"create": {"_index": "%s"}}\n' % index_name
+        if use_create and conflicting_ids:
+            raise exceptions.RallyError("'use_create' be True with 'conflicting_ids'")
```

Comment:
This error message could be a bit more user friendly. Also you mean "needs to be"?

Reply:
"Index mode '_create' cannot be used with conflicting ids"